FreeRTOS/FreeRTOS
# Add Full Demo to Qemu mps2

Add Full Demo to Qemu mps2

Description
-----------
Add Full Demo support to Qemu MPS2 with Cortex M3

Test Steps
-----------
manual testing: 
$ sudo  ~/Worskspace/qemu/build/arm-softmmu/qemu-system-arm -machine mps2-an385 -cpu cortex-m3 -kernel /media/psf/Home/workspace/FreeRTOS/FreeRTOS/Demo/CORTEX_M3_MPS2_QEMU_GCC/build/RTOSDemo.axf -netdev tap,id=mynet0,ifname=virbr0-nic,script=no -net nic,macaddr=52:54:00:12:34:AD,model=lan9118,netdev=mynet0 -object filter-dump,id=tap_dump,netdev=mynet0,file=/tmp/qemu_tap_dump -display gtk -m 16M  -nographic -serial stdio -monitor null -semihosting -semihosting-config enable=on,target=native -s 
OK: No errors - tick count 10025 
OK: No errors - tick count 20015 
OK: No errors - tick count 30015 
OK: No errors - tick count 40015 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
